plugins/module_utils/role_utils.py

```diff
@@ -42,3 +42,40 @@ def _do_subjects_differ(a, b):
     sorted_a = sorted(a, key=lambda x: (x['type'], x['name']))
     sorted_b = sorted(b, key=lambda x: (x['type'], x['name']))
     return sorted_a != sorted_b
+
+
+def _rule_sets(rules):
+    rule_set = set()
+    for rule in rules:
+        verbs = rule.get('verbs', []) or []
```

Comment:
Is there a reason for that trailing `or []`? I thnk `get` should have us covered here.